5.5/root/usr/bin/run-mysqld-slave

```diff
@@ -31,31 +25,34 @@ envsubst < ${CONTAINER_SCRIPTS_PATH}/my-paas.cnf.template > /etc/my.cnf.d/paas.c
 envsubst < ${CONTAINER_SCRIPTS_PATH}/my-slave.cnf.template > /etc/my.cnf.d/slave.cnf
 envsubst < ${CONTAINER_SCRIPTS_PATH}/my-tuning.cnf.template > /etc/my.cnf.d/tuning.cnf
 
-# Initialize MySQL database and wait for the MySQL master to accept
-# connections.
-initialize_database "$@"
-wait_for_mysql_master
+if [ ! -e "${MYSQL_DATADIR}/mysql" ]; then
```

Comment:
> so if this file does exist, (because it's a restart) the pod will start up and join as a normal slave?

If it's a restart, then the directory DOES exist.

In any case, we should always start a container with entrypoint "run-mysql-slave" as a slave, see https://github.com/openshift/mysql/pull/151#discussion_r64735585.